required/latex-lab/latex-lab-block.dtx

```diff
@@ -2018,7 +2040,11 @@
       }
     \skip_set_eq:NN \@outerparskip \parskip
     \skip_set_eq:NN \parskip \parsep
-    \dim_set_eq:NN \parindent \listparindent
+%    \end{macrocode}
+%  \changes{v0.8w}{2024/11/23}{Use private storage bin for parindent
+%    and not \cs{listparindent} to decouple block and list templates (tagging/767)}
+%    \begin{macrocode}
+    \dim_set_eq:NN \parindent \l_@@__parindent_dim
```

Comment:
ditto
```suggestion
    \dim_set_eq:NN \parindent \l_@@_parindent_dim
```

Reply:
ouch ... bad query and replace

Reply:
The previous register name was recorded in two `.tlg` files, which now failed the CI.

- `required/latex-lab/testfiles-block/blocks-000.tlg`
- `required/latex-lab/testfiles-block/blocks-000.luatex.tlg`

I didn't suggest changes in `.tlg` because it's better to not change them manually...

Reply:
sure I was aware of that and was generating them locally (accepting the temporary failure)

Reply:
and thanks as usual!